app/views/take_whole_pot_calculator/show.html.erb

```diff
@@ -0,0 +1,83 @@
+<% content_for(:data_layer) do %>
```

Comment:
My assumption was that we'd push the values into the dataLayer via JS but this could be a better approach